src/services/codefixes/importFixes.ts

```diff
@@ -823,6 +871,23 @@ function createExistingImportMap(checker: TypeChecker, importingFile: SourceFile
             const importKind = getImportKind(importingFile, exportKind, compilerOptions);
             return matchingDeclarations.map(declaration => ({ declaration, importKind, symbol, targetFlags }));
         },
+        getImportsForNonExportedSymbols: ({ symbol, importKind, targetFlags, targetFilePath }: SymbolInfo): readonly FixAddToExistingImportInfo[] => {
+            let fix: FixAddToExistingImportInfo[] = [];
+            const values = importMap?.values();
+            Debug.assertIsDefined(values);
+            for (const value of values) {
+                value.forEach(v => {
+                    if (v.kind === SyntaxKind.ImportDeclaration && cast(v.moduleSpecifier, isStringLiteral).text === targetFilePath) {
+                        fix = [{ declaration: v, importKind, targetFlags, symbol }];
+                        return fix;
+                    }
+                    else {
+                        return emptyArray;
+                    }

```

Comment:
There's no point to returning, the return value is never witnessed and it doesn't make `forEach` stop traversing.

On that note, was it your intention to stop at the first value? Because the last value will always win out in writing to `fix` in this code.

Reply:
Removed this part now.